src/content/docs/index.mdx

```diff
@@ -34,17 +34,17 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 		The modular build system can easily be extended by just writing a shell script.
     Modules you create can be added into the official BlueBuild repository or your own for the community to take advantage of.
 	</Card>
-	<Card title="Based on standards" icon="puzzle">
-		The format of your OS images is the familiar OCI image format, famously used by Docker and Podman.
+	<Card title="Based on solid standards" icon="puzzle">
+		The format of your OS images is the familiar OCI image format, famously used by Docker & Podman.
     With Fedora's Native Container support, it becomes possible to boot natively into a OCI compatible image.
 	</Card>
-	<Card title="Free and open source" icon="open-book">
-		BlueBuild is an open source project built by the community for the community.
-    Feel free to peek at and tinker with the source code!
-    GitHub also provides CI/CD and a container registry for free, though other providers are supported as well.
+	<Card title="Free & open-source" icon="open-book">
+		BlueBuild is an open-source project built by the community for the community.
+    Feel free to peek at & tinker with the source code!
+    GitHub also provides CI/CD & a container registry for free, though other providers are supported as well.
 	</Card>
-  <Card title="Effortless stability" icon="information">
-		With your custom images being based on stable atomic distributions, your setup will be rock solid.
+  <Card title="Effortless top-notch reliability" icon="information">
```

Comment:
The double-adjective sounds clunky to me. Otherwise it's fine, I'm just not sure on the *sound* of reliability over stability, but I agree it's more accurate.

Reply:
I think that double-adjective looks better than `top notch` separated.
When you just say `Effortless reliability`, it doesn't say much. It's important to indicate what kind of reliability are we talking about.

Stability is maybe a bit more catchy phrase, but I prefer to be more accurate then to be fancy.
`reliability` still sounds professional imo

Reply:
I'm not saying top-notch should be separated, but that including it sounded odd to me. If the other card titles were as long, I'd probably rewrite this to something like "Achieve top-notch reliability effortlessly"

Reply:
> I'm not saying top-notch should be separated, but that including it sounded odd to me. If the other card titles were as long, I'd probably rewrite this to something like "Achieve top-notch reliability effortlessly"

I will edit this out & think how to solve this `reliability` vs `stability` thing
Is that alright?

Reply:
Reliability is good IMO.